presto-spi/src/main/java/com/facebook/presto/spi/tracing/TracerProvider.java

```diff
@@ -19,5 +19,5 @@
      *
      * @return A @Tracer that should be kept throughout the whole duration of tracing.
      */
-    Tracer getNewTracer();
+    Tracer getNewTracer(String tracerName, String traceToken);
```

Comment:
not sure we need `tracerName` here? Can we hardcode it in the new tracer? We only need `token`